### ISSUE TITLE
Exception when using latest version of chessground

### ISSUE THREAD

**Issue by vjuneja** [ruilisi/react-chessground#23]
This exception started happening with version `7.8.0` of  `chessground`. For a temporary fix, I have hardcoded my yarn.lock file to version `chessground@7.7.2`. 

Most likely cased by this change. https://github.com/ornicar/chessground/commit/c190c1823f5836aebc156a248289fa300a248675#diff-a2fcebf1963fad0da40a844a8d67da83R48

Creating an issue on `react-chessground` because I am not sure if the root cause is the react component not sending some info to the `chessground` component.

Thanks.

PS: this is reproducible in the demos if you try to rebuild the demo projects by deleting yarn.lock. I tried it with the `example/demo` project.

```
TypeError: _a.get is not a function
computeSquareClasses
src/render.ts:221
render
src/render.ts:16
  13 | if (isPieceNode(el)) {
  14 |     pieceAtKey = pieces.get(k);
  15 |     anim = anims.get(k);
> 16 |     fading = fadings.get(k);
     | ^  17 |     elPieceName = el.cgPiece;
  18 |     if (el.cgDragging && (!curDrag || curDrag.orig !== k)) {
  19 |         el.classList.remove('dragging');
View compiled
redrawNow
src/chessground.ts:25
  22 |     bounds.clear();
  23 |     render_1.updateBounds(state);
  24 |     if (elements.svg)
> 25 |         svg.renderSvg(state, elements.svg);
     | ^  26 | };
  27 | state.dom = {
  28 |     elements,
```

![Screen Shot 2020-07-19 at 2 51 08 PM](https://user-images.githubusercontent.com/13910458/87886142-79607800-c9cf-11ea-8d2c-698efe5cd538.png)

**Comment by vjuneja** [comment]
based on https://github.com/ornicar/chessground/issues/148
the fix needs to be on `react-chessground`

**Comment by fybchristasker** [comment]
HI, this will be fixed in  new version, temporary, its better to use older version of chessground.

**Comment by vjuneja** [comment]
There is only one version of `react-chessground` available on npm js and that is `1.5.0` which is the broken one. Up to you how you want to handle this. It is better to keep the issue open until this is resolved in case someone comes looking for the workaround.

**Comment by Roybie** [comment]
Does anyone have a solution for this? Why is it closed?
